shepherd_bms/include/bmsConfig.h

```diff
@@ -1,16 +1,37 @@
-#ifndef BMS_COMFIG_H
+#ifndef BMS_CONFIG_H
```

Comment:
im not sure if this is valid or not but if there's going to be a clear difference between known/confirmed macros vs values that we are still tweaking and tuning, maybe worth commenting to note which ones are which. Would be nice to know how many of these are still being tested vs which ones should never need to be touched

Reply:
I think we should mark out which ones are purely for safety/shutdown and which ones can impact performance. But anything that can impact performance will likely never be finished. Always an evolution